stable/admin/values.yaml

```diff
@@ -18,7 +18,13 @@ cloud:
   # cluster name is used to make resources unique in multi-cluster configurations.
   # If the NuoDB domain spans 2 or more physical clusters, then each cluster must have a unique clusterName.
   # The default is fine for single-cluster domains.
-  clusteName: cluster0
+  clusterName: cluster0
```

Comment:
As per first comment, I would suggest that we have enough `cluster` values that it now warrants a separate `cluster` section.